examples/node/v12/__tests__/relationships.test.ts

```diff
@@ -0,0 +1,324 @@
+import Realm from "realm";
```

Comment:
Similar to #2980, let's use top-level named imports as that's a new feature in v12 and how most TypeScript/JavaScript devs prefer to work.
```suggestion
import Realm, {BSON} from "realm";
```

Reply:
Then, we can replace all instances of `Realm.BSON` with just `BSON`.

Reply:
Thanks for catching this! I meant to update it after I updated the other PR, but got pulled away and had forgotten by the time I got back to it.